### ISSUE TITLE
Chile2010 example download location has a redirect

### ISSUE THREAD

**Issue by cjeffr** [clawpack/geoclaw#385]
When installing geoclaw on a new computer I discovered that the chile2010 example was getting a 'topo arrays do not cover domain' error.
```                                                                         
 **** topo arrays do not cover domain
 **** area of overlap =    100.00000000000597     
 **** area of domain  =    3600.0000000000000     
Note: The following floating-point exceptions are signalling: IEEE_UNDERFLOW_FLAG IEEE_DENORMAL
```
While investigating I found that the download location in the maketopo.py is actually being redirected from http://www.geoclaw.org/topo/etopo/ to 
http://depts.washington.edu/clawpack/geoclaw/ and thus downloaded that page rather than the etopo file. I've fixed this in my version but wanted to ensure someone else saw this so it could be fixed officially. I've attached my fixed maketopo.py

[maketopo.py.txt](https://github.com/clawpack/geoclaw/files/3192941/maketopo.py.txt)

**Comment by rjleveque** [comment]
Sorry about the broken redirect!  I also noticed this recently, and it will be fixed in the next release but that got delayed a bit with sorting out some other things...

In the meantime, it is fixed in the master branch in the same manner as in your version, or other can simply replace the offending line in `maketopo.py`:
```
    url = 'http://www.geoclaw.org/topo/etopo/' + topo_fname
```
with the corrected version:
```
    url = 'http://depts.washington.edu/clawpack/geoclaw/topo/etopo/' + topo_fname
```

**Comment by rjleveque** [comment]
Fixed as of v5.6.1.
